entries/c/cloud.baidu.com.json

```
@@ -0,0 +1,17 @@
+{
+    "Baidu AI cloud": {
+      "domain": "cloud.baidu.com",
```

The authentication seems to be done at a separate domain.
```suggestion
      "domain": "cloud.baidu.com",
      "additional-domains": [
        "login.bce.baidu.com"
      ],
```